src/SmartComponents/__tests__/completedTaskDetailsHelpers.tests.js

```diff
@@ -0,0 +1,97 @@
+import {
+  getSelectedSystems,
+  isError,
+  fetchTask,
+  fetchTaskJobs,
+} from '../completedTaskDetailsHelpers';
+import { log4j_task_jobs } from '../CompletedTaskDetails/__tests__/__fixtures__/completedTasksDetails.fixtures';
+import {
+  errorResponse,
```

Comment:
```
export const isError = (result) => {
  return result?.response?.status && result?.response?.status !== 200;
};
```

201, 202, 203, 204, 205, 206 response codes be like:
![Untitled-5](https://user-images.githubusercontent.com/8426204/185469665-e75a8537-e24c-4880-8423-63f6c3c3fe11.jpg)

I know the API probably uses 200 only, but let's futureproof this by checking for the first digit :smile: